scripts/add_nonce.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+# This must be called from repo root as current directory
+
+NONCE=$(date +%s)
+
+for packagedir in ./packages/*; do
+  if [[ -d "$packagedir" ]]; then
+    echo "$packagedir"
+    mkdir -p "$packagedir/nonces"
+    echo "Directory used to trigger lerna package updates for all packages" \
+      > "$packagedir/nonces/README.txt"
+    touch "$packagedir/nonces/$NONCE"
```

Comment:
Do we also want to remove the old nonce(s)? Alternatively we could just have a file which changes.

Reply:
The idea here is that multiple files in a directory can be merged automatically. This saves time when merging e.g. the branch 0.12 into master when there are 0.12.x releases (and potentially 0.13.x on master)